libs/ui/src/lib/form-message/form-message.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  HostBinding,
+} from '@angular/core';
+import { CviIconName } from '@egov/cvi-icons';
+import { FormMessageSeverity } from './form-message';
+import { FormMessageSeverityToHeaderIconPipe } from './form-message-severity-to-header-icon.pipe';
+
+@Component({
+  selector: 'cvi-ng-form-message',
+  templateUrl: './form-message.component.html',
+  providers: [FormMessageSeverityToHeaderIconPipe],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FormMessageComponent {
+  message = '';
+  display!: boolean;
+  severity!: FormMessageSeverity;
+  errorLabel!: string;
```

Comment:
The name is a bit confusing – we have a value for `severity` prop named `error`, so a developer might think that this prop somehow only sets a label when `severity="error"`. Simply `label` or `title` would work better.